src/main/webapp/app/core/icons/font-awesome-icons.ts

```diff
@@ -116,6 +116,7 @@ import {
     faUserCheck,
     faUserMinus,
     faUserPlus,
+    faUserSlash,
```

Comment:
I am not quite sure if the warning is accurate here as the `programming-exercise-lifecycle.component.html` uses the icon `<fa-icon [icon]="'user-slash'" size="lg"></fa-icon>` and as far as I know this is the way to import new icons. Am I mistaken there? Otherwise I would ignore this warning.

Reply:
ok looks like i missed that one. Please ignore my comment.